docs/changelog.txt

```diff
@@ -1,3 +1,26 @@
+0.5.1 (2020-05-10)
+------------------------------------------------------------------------
+Among other, debian packaging related reasons, this release firms up the number of allowed Road/Tramtypes.
+
+Roadtypes
+---------
+According to the NFO spec, there can be up to 63 each of road/tramtypes, rather than the 15 each that were allowed previously.
+OpenTTD also requires that the *combined* total of road and tram types be no more than 63, and that a label used for a roadtype cannot be used for a tramtype or vice versa.
+NML will not check for, nor enforce, these additional limitations.
+
+Other changes
+-------------
+- Add: Builtin functions roadtype() and tramtype()
+- Fix #116: TownNames used bits could be the same for sub-parts (#117)
+- Fix #120: Value range of transported_last_month_pct should be 0..100 (#122)
+- Fix #121: STORE_PERM address limit should be 63, not 15
+- Fix #134: Incorrect value of PALETTE_CC_RED
+- Fix #74: Update ottd_display_speed to match changes from OpenTTD r23945 (#75)
+- Fix: ANIM warnings in examples
+- Fix: Correct limit of available road/tramtypes
+- Fix: Produce, TownNames & TracktypeTable debug_print (#126, #127, #129)
```

Comment:
Fix: Don't attempt to determine version from git when building from a release tarball.